README.md

```diff
@@ -1,27 +1,38 @@
 
 ## Jekyll Installation
-Swap out `rbenv` below for `rvm` if you prefer. RVM was giving me installation issues, so I found `rbenv` - Travis
+Jekyll runs on Ruby, and running multiple versions of Ruby on a single system is apparently difficult. These instructions install [`rbenv`](https://github.com/rbenv/rbenv), which makes it easy to install and switch to a specific Ruby version.
 
-1. Create and run from a fresh VM instance:
-    1. `gcloud compute instances create jekyll --image-project=ubuntu-os-cloud --image-family=ubuntu-1404-lts --machine-type=n1-standard-1`
+1. (Optional) Create and run from a fresh VM instance, and port forward the Jekyll default port (4000):
+    1. `gcloud compute instances create jekyll --image-project=ubuntu-os-cloud --image-family=ubuntu-1804-lts --machine-type=n1-standard-1`
     1. `gcloud compute ssh jekyll --ssh-flag="-L 4000:localhost:4000"`
 1. Install `rbenv` and `ruby-build`. Add these to `$PATH`:
-    1. `sudo apt-get install -y git bzip2 build-essential libssl-dev libreadline-dev zlib1g-dev`
-    1. `git clone https://github.com/rbenv/rbenv.git ~/.rbenv`
-    1. `git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build`
-    1. `echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc`
-    1. `echo 'eval "$(rbenv init -)"' >> ~/.bashrc`
-    1. `. ~/.bashrc`
+    ```
+    sudo apt update
+    sudo apt-get install -y git bzip2 build-essential libssl-dev libreadline-dev zlib1g-dev
+    git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+    git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+    echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+    echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+    source ~/.bashrc
+    ```
 1. Install and use ruby 2.4.1
-    1. `rbenv install 2.4.1`
-    1. `rbenv global 2.4.1`
+    ```
+    rbenv install 2.4.1
+    rbenv global 2.4.1
+    ```
 1. Fork and clone your forked repo:
-    1. `GITHUB_USER=$USER # or something else here`
-    1. `git clone https://github.com/$GITHUB_USER/spinnaker.github.io.git`
+    ```
+    GITHUB_USER=$USER # or something else here
+    git clone https://github.com/$GITHUB_USER/spinnaker.github.io.git
+    ```
 1. Install `bundle` gem
-    1. `cd spinnaker.github.io`
-    1. `gem install bundler`
-    1. `bundle install`
+    ```
+    cd spinnaker.github.io
+    gem install bundler
+    bundle install
+    ```
+
+ Your system should now be ready for local development.
```

Comment:
```suggestion
 Your system is now ready for local preview of the documentation site.
```

Reply:
Changed from "development" to avoid sounding like we're talking about local _Spinnaker_ development.

Reply:
Done.